case-studies/bayesline.md

```diff
@@ -41,34 +45,36 @@ Why Bayesline chose Reflex instead of Plotly Dash
 
 ## TLDR;
 
-Sebastian and his small team use Reflex to build a production-grade web app while still keeping the full flexibility of a traditional ReactJS app **WITHOUT** needing to:
+Sebastian and his small team use Reflex to build a production-grade web app purely in Python while still keeping the full flexibility of a traditional ReactJS app **WITHOUT** needing to:
 
-* Learn frontend: React (JavaScript), NodeJS, TailwindCSS
+* Learn frontend technologies: React (JavaScript), NodeJS, TailwindCSS
 
 ```md quote Misha and Sebastian
 Reflex definitely saved us from needing to hire a frontend engineer and sped us up by 4x relative to learning React
 ```
 
 * Write boilerplate to stitch together their frontend and backend (including database management)
 
-```md quote Sebastian
-50% less code than the same Dash app and easier to easier to read / write / maintain code compared to Dash
+```md quote Sebastian CTO
+50% less code than the same Dash app and easier to read / write / maintain code compared to Dash
 ```
 
 * Maintain expensive, fragile, and inevitably slow Dash apps
 
-```md quote Misha
+```md quote Misha CEO
 Using Reflex instead of Plotly Dash was like the difference between organized Legos and a plate of spaghetti
 ```
 
-## What Baylesline is building
+## What Bayesline is building
```

Comment:
I think what what Bayesline is building should come first then  TLDR a the end as a summary, with a new title 
"What bayesline gained from using Reflex"

Reply:
Since we have that stats cards acting as a tldr at the top

Reply:
added this